microblaze/anal/anal_microblaze_gnu.c

```diff
@@ -19,9 +19,9 @@
 #define get_field_imm(instr)       get_imm   (ctx, instr)
 
 struct mb_anal_ctx {
-	int immval;
+	unsigned immval;
```

Comment:
Unsigned wat. I know this compiles but its not clear because it doesnt specify the size of it. Use ut32 or ut64 instead